### PR TITLE
Fix Egg Bug

### PR DESCRIPTION
**Pull request by Koalinrain**
fix: fix http response bug on egg

 skywalking cannot logging while node send request with egg framework

### CODE REVIEW

**Comment by wu-sheng** [comment]
Egg seems a Chinese local community mostly. I am not sure whether @tom-pytel has chance to get touch of it.

**Review by kezhenxu94 — APPROVED** [review]
> Egg seems a Chinese local community mostly. I am not sure whether @tom-pytel has chance to get touch of it.

Once the tests passed I'm good to merge this, unless @tom-pytel has something want to add.

**Comment by wu-sheng** [comment]
No update in one month. Closing. @Koalinrain If you want back on this, please let us know.

**Pull request closed without merging**
